lib/pool.js

```diff
@@ -33,15 +25,14 @@ function Pool(url, options = {}) {
   )
 }
 
-Pool.prototype.request = function (method, path, data) {
+Pool.prototype.request = function (method, path, extraHeaders, data) {
   var d = P.defer()
+  var headers = { 'Content-Type': 'application/json' }
```

Comment:
Last drive-by comment, I swear... And [of course] ESLint has a pair of rules for this too, if you want to enforce it at a linter level.

- [**no-var**](https://eslint.org/docs/rules/no-var): Require `let` or `const` instead of `var`. (Currently 1575 errors/warnings)
- [**prefer-const**](https://eslint.org/docs/rules/prefer-const): Suggest using `const` if a variable is never reassigned. (Currently 0 errors/warnings)